src/test/resources/outputs/redshift/operators/between.sql

```diff
@@ -1,23 +1,23 @@
 --#[between-00]
 -- between(decimal, int32, int32)
-SELECT "T_DECIMALS"."da" FROM "default"."T_DECIMALS" AS "T_DECIMALS" WHERE "T_DECIMALS"."da" BETWEEN CAST(-1 AS DECIMAL) AND CAST(1 AS DECIMAL);
+SELECT "T_DECIMALS"."da" FROM "default"."T_DECIMALS" AS "T_DECIMALS" WHERE "T_DECIMALS"."da" BETWEEN -1 AND 1;
```

Comment:
self-review: the casts of int to decimal were redundant here as Redshift supports `BETWEEN` with mixed numeric arg types (int + decimal) without explicit casts